<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Floppy Disk Emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="FloppyDiskWidget" name="floppyWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="FDCControllerWidget" name="fdcWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
      </property>
      <property name="minimumHeight">
       <number>200</number>
      </property>
      <property name="maximumHeight">
       <number>200</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionToggleView"/>
   <widget class="QLabel" name="speedLabel">
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="speedComboBox">
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>0.01x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.05x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.1x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.25x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.5x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2x</string>
     </property>
    </item>
   </widget>
  </widget>
  <action name="actionPlay">
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop Animation</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Animation</string>
   </property>
  </action>
  <action name="actionToggleView">
   <property name="text">
    <string>Front / Back</string>
   </property>
   <property name="toolTip">
    <string>Toggle between Front and Back views</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloppyDiskWidget</class>
   <extends>QWidget</extends>
   <header>floppydiskwidget.h</header>
  </customwidget>
  <customwidget>
   <class>FDCControllerWidget</class>
   <extends>QWidget</extends>
   <header>fdccontrollerwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui> 